importexport/musicxml/exportxml.cpp

```diff
@@ -3345,7 +3345,6 @@ static QString elementPosition(const ExportMusicXml* const expMxml, const Elemen
             const double pageHeight  = expMxml->getTenthsFromInches(expMxml->score()->styleD(Sid::pageHeight));
 
             const auto meas = elm->findMeasure();
-            Q_ASSERT(!meas);
```

Comment:
@lvinken Any idea what this was good for and why I'd need to remove it for this fix not to crash in Debug builds?

Reply:
Hi Jojo, it is somewhat late at night, but so far I don't understand. My 3.x (commit 1b3ed3184 by RomanPudashkin on Wed May 5 10:14:03 2021 +0200) does not have either elementPosition or Q_ASSERT(!meas). Instead it has a similar function static QString notePosition(const ExportMusicXml* const expMxml, const Note* const note), which  does not contain any assertions. And, no, I have no idea why the assertion would be useful. Could you check the source and history of your exportxml.cpp ?

Reply:
See https://github.com/musescore/MuseScore/pull/9955#issuecomment-986015549, seems my fault, a bad backport.